### DISCUSSION TITLE
Collapse (don't expand) nested brackets

### DISCUSSION THREAD

**Discussion by jsh9** [jsh9/cercis#14] [General]
Hi @thinkwelltwd, @jni, @strokirk, @kevina, @bmreiniger:

I have been working on a more configurable Python code formatter (at least more configurable than Black) and came across this post: https://github.com/grantjenks/blue/issues/84

You either wrote it or voted up.

My tool (Cercis) offer this style by default.  See: https://github.com/jsh9/cercis#36-collapse-nested-brackets

Please feel free to try it out, and please feel free to share feedback (either here or in Issues).  Thanks.

**Comment by jni** [comment]
Sweet mercy 😭 🙏

**Comment by strokirk** [comment]
![they-do-come-with-configurable-options](https://user-images.githubusercontent.com/1257824/236863138-7f89573e-ad5c-459f-8e14-98ca690287da.gif)

_They DO come with configurable options...!_
